### ISSUE TITLE
Failed to resolve: com.github.blongho:worldCountryData:1.5.3

### ISSUE THREAD

**Issue by ksgangadharan** [blongho/worldCountryData#45]
The 1.5.3 pom file seems to be missing in jitpack. The path is" `https://jitpack.io/com/github/blongho/worldCountryData/1.5.3/worldCountryData-1.5.3.pom`.  I can see the 1.5.2 files there, but no 1.5.3.  Problem started lately I guess.

**Comment by blongho** [comment]
@ksgangadharan do you still experience the same issue?

**Comment by ksgangadharan** [comment]
@blongho yes, if you check the path in the browser, you will see that its not present. We have since changed to another library, so no longer tracking the issue.

**Comment by PiotrPrus** [comment]
Hey, I can confirm it is not working with 1.5.4-alpha1 as well. Same issue. The file is not present.

**Comment by blongho** [comment]
I will take a look at this issue in the days ahead. In the mean time, can you revert to the previous version while I figure out what went wrong? @PiotrPrus  @ksgangadharan

**Comment by blongho** [comment]
> Hey, I can confirm it is not working with 1.5.4-alpha1 as well. Same issue. The file is not present.
@PiotrPrus @ksgangadharan 
I have tried to build the application with v1.5.4-alpha-1 and it works well. See my settings at [Using v1.5.4-alpha-1](https://github.com/blongho/worldCountryData/issues/47#issuecomment-2176479948)
